### DISCUSSION TITLE
Can I delete the template for the vulnerability CVE-2022-46463?

### DISCUSSION THREAD

**Discussion by qingchenhh** [projectdiscovery/nuclei-templates#8964] [Q&A]
I don't think CVE-2022-46463 is a valid vulnerability.

**Comment by gtrrnr** [comment]
Doubt that deleting it will be a good idea.
You can however use:
```
   -eid, -exclude-id string[]         templates to exclude based on template ids (comma-separated, file)
```
To exclude templates that are prone to false positives or have questionable impact.

**Comment by olearycrew** [comment]
@qingchenhh I agree with @gtrrnr on how you can exclude the template, but can you also share what problem you're having with this particular template? I haven't seen many false positives.

**Comment by princechaddha** [comment]
@qingchenhh, it appears to be a valid CVE since it leaks repository details without requiring authentication. Could you share any debug data if you are encountering false positive results? Alternatively, if you wish to ignore it, you can add it to your Nuclei configuration at `~/.config/nuclei/config.yaml`

**Reply by qingchenhh** [reply]
No, I don't think it's an effective vulnerability because it can only search for public repositories. Do you think public repository leaks belong to information leaks?
